packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -1171,20 +1175,22 @@ String prettyTimestamp(
 /// listeners are notified whenever the data is modified, but notifying can be
 /// optionally disabled.
 class ListValueNotifier<T> extends ChangeNotifier
-    implements ValueListenable<List<T>> {
+    implements ValueListenable<List<T>?> {
   /// Creates a [ListValueNotifier] that wraps this value [_rawList].
   ListValueNotifier(List<T> rawList) : _rawList = List<T>.from(rawList) {
     _currentList = ImmutableList(_rawList);
   }
 
   List<T> _rawList;
 
-  ImmutableList<T> _currentList;
+  ImmutableList<T>? _currentList;
```

Comment:
I think this can be late instead of nullable

Reply:
done